AssertionError in metrics.py when searching

The following search causes an `AssertionError`.

`$ grascii search -g n-dv`

Output:

```
(Token('DV', 'DV'), set())
Traceback (most recent call last):
  File "bin/grascii", line 8, in <module>
    sys.exit(main())
  File "lib/python3.9/site-packages/grascii/__main__.py", line 37, in main
    args.func(args)
  File "lib/python3.9/site-packages/grascii/search.py", line 105, in cli_search
    results = search(**{k: v for k, v in vars(args).items() if v is not None})
  File "lib/python3.9/site-packages/grascii/search.py", line 97, in search
    return searcher.search(**kwargs)
  File "lib/python3.9/site-packages/grascii/searchers.py", line 154, in search
    return list(results)
  File "lib/python3.9/site-packages/grascii/searchers.py", line 54, in perform_search
    diff = min(diff, metric(interp, match))
  File "lib/python3.9/site-packages/grascii/metrics.py", line 204, in standard
    return match_distance(g1, g2)
  File "lib/python3.9/site-packages/grascii/metrics.py", line 180, in match_distance
    v1[0] = v0[0] + compute_ins_del_cost(g1[i])
  File "lib/python3.9/site-packages/grascii/metrics.py", line 118, in compute_ins_del_cost
    assert cost > 0
AssertionError
```

Oops, looks like `grammar.STROKES` is also missing `TV`.

```
grascii search -g df
```

```
  File "grascii/grascii/metrics.py", line 86, in compute_ins_del_cost
    assert cost > 0, stroke
AssertionError: AnnotatedStroke(stroke='TV', annotations=set())
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -7,6 +7,10 @@
 
 - Some classes and functions that are considered to be part of the public API are importable from the top-level `grascii`.
 
+### Fixed
+
+- Included `TV` in `grammar.STROKES`.
+
 ## 0.4.0 - 2022-06-27
 
 ### Added

--- a/grascii/grammar.py
+++ b/grascii/grammar.py
@@ -30,7 +30,7 @@
 from typing import Dict, List, Tuple
 
 _STROKES = "A AU A&' A&E B CH D DD DF DM DN DT DV E EU F G I J JND JNT K L LD \
-    M MD MM MN MT N ND NG NK NT O OE P PND PNT R S SH SS T TD TH TM TN U V X XS Z"
+    M MD MM MN MT N ND NG NK NT O OE P PND PNT R S SH SS T TD TH TM TN TV U V X XS Z"
 _VOWELS = "A AU A&' A&E E EU I O OE U"
 
 STROKES = set(_STROKES.split())

